pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -303,6 +303,13 @@ func (e *eksOperatorController) onClusterChange(key string, cluster *mgmtv3.Clus
 			}
 		}
 
+		clusterLaunchTemplateID := status["launchTemplateID"].(string)
```

Comment:
```suggestion
		clusterLaunchTemplateID := status[EKSStatusFieldLaunchTemplateID].(string)
```

Reply:
Technically, these are different fields. The `EKSStatusFieldLaunchTemplateID` refers to a field of the `cluster` object, but we are looking for a field of the `eksclusterconfig` object.